aregion.cpp

```diff
@@ -103,11 +103,13 @@ TownInfo::~TownInfo()
 	if (name) delete name;
 }
 
-void TownInfo::Readin(Ainfile *f, ATL_VER &v)
```

Comment:
the ATL_VER was being passed down to *some* but not all of these and never used.. I pulled it out since it wasn't needed.